pkg/controller/operators/olm/operator.go

```diff
@@ -346,6 +346,22 @@ func (a *Operator) handleClusterServiceVersionDeletion(obj interface{}) {
 	defer func(csv v1alpha1.ClusterServiceVersion) {
 		logger.Debug("removing csv from queue set")
 		a.csvQueueSet.Remove(csv.GetName(), csv.GetNamespace())
+		
+		// Requeue all OperatorGroups in the namespace
```

Comment:
While I was working on this, I decided to make OperatorGroup requeuing on CSV deletion more robust.